task_spec

```diff
@@ -0,0 +1,90 @@
+1st Protocol Feature `stop_deferred_transactions`
```

Comment:
This file does not belong here.  It's also inconsistent with the PR description.

Reply:
File has been removed.